bin/GameIndex.dbf

```diff
@@ -124,6 +124,7 @@ Region = PAL-Unk
 Serial = PBPX-95503
 Name   = Gran Turismo 3 - A-Spec [PS2 Bundle]
 Region = NTSC-U
+MemCardFilter = PBPX-95503/SCUS-97102
```

Comment:
I left it in as a precaution, just in case it does at some point write something to a save corresponding to its own serial. Can take it again if you think it's unnecessary. Personally I'd err on the side of caution.